api/README.md

```
@@ -138,16 +138,16 @@ vim application/usecases/spot_usecase.go
 vim interfaces/http/handlers/spot_handler.go
 ```
 
-### 2. Testing Strategy
+### 2. BDD Testing with Ginkgo
 ```bash
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)